### PR TITLE
WDPT-REMOTE-SEP-Michael-Seitz

### PR DESCRIPTION
**Pull request by Gk1mtd**

### CODE REVIEW

File: src/functions-and-arrays.js [hunk 1]
```diff
@@ -1,41 +1,168 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(numA, numB) {
+  let largestNumber
```
Comment (tomasyaya) [inline src/functions-and-arrays.js:3]: You could do

`return Math.max(numA, numB)`

File: src/functions-and-arrays.js [hunk 2]
```diff
@@ -1,41 +1,168 @@
...
 
 // Iteration #3.1 Bonus:
-function sum() {}
+function sum(numbers) {
+  if (numbers.length === 0) { //returns 0 if array is empty
+    return 0
+  }
+
+  for (element of numbers) {
+    if (typeof element === "object")
```
Comment (tomasyaya) [inline src/functions-and-arrays.js:65]: if you use a different loop just for the throw path then you are going to loop the array twice, you could add it to the other loop

**Review by tomasyaya — COMMENTED** [review]
Great work with this LAB @Gk1mtd

File: src/functions-and-arrays.js (hunk 1 above)
Comment (Gk1mtd) [reply src/functions-and-arrays.js:3]: :D that will come in handy! Thanks.

**Pull request closed without merging**